tests/test_api_keys.py

```diff
@@ -0,0 +1,56 @@
+from app.db import Session
```

Comment:
I think it'd be better to test the actual action of clicking the "delete all api keys" button. This can be simulated by sending a Post request, you can take a look at https://github.com/simple-login/app/blob/38ecb227b0fc51c237c2e0baead6ac52b96ea21c/tests/dashboard/test_directory.py#L1-L0 for an example.

Reply:
Done